doc/getting-started.adoc

```diff
@@ -64,7 +69,7 @@ $ cd nixpkgs
 $ git checkout $revision
 $ cd ~/Projects/mobile-nixos/
 $ export NIX_PATH="nixpkgs=$HOME/Projects/nixpkgs"
-$ nix-build [...]
+$ nix-build --arg pkgs 'import /home/user/Projects/nixpkgs {}' [...]
```

Comment:
that seem to be duplicating when you have set `NIX_PATH`

please remove ... and clarify what command to run. as a new user i can't imagine what to put there...